content-repo/extra-docs/releases/21.3.1.md

```diff
@@ -0,0 +1,898 @@
+- Updated the Docker image to: *demisto/python3:3.9.1.15759*.
+
+---
+
+### Common Playbooks Pack v1.9.0
+#### Playbooks
+##### Get File Sample By Hash - Generic v3
+- Added the Code42 File Download as a subplaybook to retrieve files using the Code42 integration.
+- Fixed an issue that caused an error in the playbook when one of the subplaybooks was not available.
+
+##### Get File Sample From Path - Generic V2
+Fixed an issue that caused an error in the playbook when one of the subplaybooks was not available.
+
+---
+
+### Common Scripts Pack v1.3.28
+#### Scripts
+##### CheckFieldValue
+Fixed an issue where the script was failing for certain fields.
+
+##### ConvertXmlToJson
+Metadata and documentation enhancements.
+
+##### IncidentAddSystem
+Metadata and documentation enhancements.
+
+##### ParseJSON
+Metadata and documentation enhancements.
+
+##### IsGreaterThan
+Metadata and documentation enhancements.
+
+##### MathUtil
+Metadata and documentation enhancements.
+
+##### CreateArray
+Metadata and documentation enhancements.
+
+##### ConvertXmlFileToJson
+Metadata and documentation enhancements.
+
+##### UnEscapeIPs
+Metadata and documentation enhancements.
+
+##### CreateEmailHtmlBody
+Metadata and documentation enhancements.
+
+##### LessThanPercentage
+Metadata and documentation enhancements.
+
+##### FileCreateAndUpload
+Metadata and documentation enhancements.
+
+##### NotInContextVerification
+Metadata and documentation enhancements.
+
+##### FindSimilarIncidents
+- Fixed an issue where the script would not find similar incidents while using the **similarIncidentFields** argument when given an int type.
+- Upgraded the Docker image to: *demisto/python:2.7.18.15765*.
+
+##### PortListenCheck
+Metadata and documentation enhancements.
+
+##### DumpJSON
+Metadata and documentation enhancements.
+
+##### UnPackFile
+Metadata and documentation enhancements.
+
+##### Base64EncodeV2
+Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
+
+##### DT
+Metadata and documentation enhancements.
+
+##### InRange
+Metadata and documentation enhancements.
+
+##### GetDuplicatesMlv2
+The script will now run on a separate container.
+
+##### IsRFC1918Address
+- Fixed an issue where the script was not working as a transformer.
+- Upgraded the Docker image to: *demisto/netutils:1.0.0.14492*.
+
+---
+
+### Common Types Pack v2.8.7
+#### Incident Fields
+**Hostnames**
+
+
+#### Indicator Types
+**urlRep**
+- Fixed an issue where URLs with an underscore were not extracted.
+
+---
+
+### ConcentricAI Pack v1.2.0 (Partner Supported)
+#### Integrations
+##### ConcentricAI
+Added enhancement to playbook and overall functioning.
+
+#### Playbooks
+##### ConcentricAI Demo Playbook
+The Playbook now enables one to fetch all file details, file sharing permissions, user details of the owner along with summaryReport of the incident.
+
+---
+
+### Cyberint Pack v1.0.3 (Partner Supported)
+#### Incident Fields
+- **incident_cyberintalertid**
```

Comment:
something is off in the formatting here, should probably just be a list of the incident fields